### PR TITLE
HTML form by MG

### PR DESCRIPTION
**Pull request by M-Grisha**
https://M-Grisha.github.io/layout_html-form/
https://M-Grisha.github.io/layout_html-form/report/html_report/

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -8,10 +8,201 @@
...
+            >
+          </label>
+        </div>
+        <div>
+          <label>
+            Full date of birth:
+            <input
+              type="number"
+              name="birth"
+              placeholder="yyyy / mm / dd"
```
Comment (Bohdan-Maksymiuk) [inline src/index.html:63]: remove

File: src/index.html [hunk 2]
```diff
@@ -8,10 +8,201 @@
...
+            </select>
+          </label>
+        </div>
+      </fieldset>
+
+      <div>
+        <button type="submit" >Submit</button>
+      </div>
+
+    </form>
```
Comment (Bohdan-Maksymiuk) [inline src/index.html:206]:
```suggestion
      <div>
        <button type="submit" >Submit</button>
      </div>
    </form>
```

File: src/style.css [hunk 3]
```diff
@@ -1 +1,11 @@
...
+  margin-bottom: 10px;
+}
+
+div:last-child {
+  margin-bottom: 0;
+}
+
+fieldset {
+  margin-bottom: 21px;
+}
```
Comment (Bohdan-Maksymiuk) [inline src/style.css:11]: use class selectors

**Review by Bohdan-Maksymiuk — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 1 above)
Comment (M-Grisha) [reply src/index.html]: Check this pleas..its my last correction
develop branch, commit - "html is realy ready"

File: src/style.css [hunk 4]
```diff
@@ -1 +1,15 @@
...
+
+.input-box:last-child {
+  margin-bottom: 0;
+}
+
+/* label {
+  display: block;
+} */
+
```
Comment (valerii-derkach) [inline src/style.css:12]: delete comments

**Review by valerii-derkach — APPROVED** [review]

**Pull request closed without merging**